t/test_validate_syntax.t

```diff
@@ -221,21 +222,24 @@ subtest 'Everything but NoWarnings' => sub {
 
     $frontend_params->{ds_info}->[0]->{algorithm} = 1;
     $frontend_params->{ds_info}->[0]->{digest}    = '0123456789012345678901234567890123456789';
-    is( $engine->validate_syntax( $frontend_params )->{status}, 'ok', encode_utf8( 'Valid Algorithm Type [numeric format]' ) )
-        or diag( $engine->validate_syntax( $frontend_params )->{message} );
+    $frontend_params->{ds_info}->[0]->{digtype}   = 1;
+    $frontend_params->{ds_info}->[0]->{keytag}   = 5000;
```

Comment:
Would you mind adding tests with `digtype` and `keytag`?

Reply:
done.